browser/main/lib/modal.js

```diff
@@ -15,9 +15,13 @@ class ModalBase extends React.Component {
 
   close () {
     if (modalBase != null) modalBase.setState({component: null, componentProps: null, isHidden: true})
-    // Toggle overflow style on NoteList
-    let list = document.querySelector('.NoteList__list___browser-main-NoteList-')
-    list.style.overflow = 'auto'
+
+    // Toggle display for scrollbars
+    let list = document.querySelectorAll('.NoteList__list___browser-main-NoteList-, .SnippetNoteDetail__description___browser-main-Detail- > textarea')
```

Comment:
I think that modal component shouldn't know about other components. It should be reusable. Same below.